submarine-server/server-submitter/submitter-k8s/src/test/java/org/apache/submarine/server/submitter/k8s/K8SJobSubmitterTest.java

```diff
@@ -62,21 +51,10 @@
  * Local: docker run -it --privileged -p 8443:8443 -p 10080:10080 bsycorp/kind:latest-1.15
  * Travis: See '.travis.yml'
  */
-public class K8SJobSubmitterTest {
-  private final String tfJobName = "mnist";
-  private final String pytorchJobName = "pytorch-dist-mnist-gloo";
-
-  // The spec files in test/resources
-  private final String tfJobSpecFile = "/tf_job_mnist.json";
-  private final String tfJobReqFile = "/tf_mnist_req.json";
-  private final String pytorchJobSpecFile = "/pytorch_job_mnist_gloo.json";
-  private final String pytorchJobReqFile = "/pytorch_job_req.json";
+public class K8SJobSubmitterTest extends SpecBuilder {
```

Comment:
It looks like we don't have tests for patchJob and listJob. We need to add them.